### ISSUE TITLE
Add support for Google nest temperature sensors

### ISSUE THREAD

**Issue by guillochon** [potmat/homebridge-google-nest-sdm#114]
Talking about these guys: https://store.google.com/product/nest_temperature_sensor?hl=en-US

Are these exposed in the SDM API?

**Comment by potmat** [comment]
Nope: https://developers.google.com/nest/device-access/supported-devices

<img width="553" alt="image" src="https://github.com/potmat/homebridge-google-nest-sdm/assets/7747977/4a8f1160-d49c-44d1-a938-18ce6c9bf9ed">

**Comment by guillochon** [comment]
Bummer

**Comment by potmat** [comment]
@guillochon: Probably the same reason smoke alarms are not supported, they are low power devices.  If they were exposed via an API they would have to be constantly updating their status which would run down the batteries inside very quickly.

**Comment by guillochon** [comment]
Any idea how the other Google Nest homebridge plugin accesses them? They are exposed there, though I have no idea how it does it. Just wondering if that's replicable.

**Comment by potmat** [comment]
They use unofficial/undocumented API calls.
